include/soci/soci-backend.h

```diff
@@ -342,7 +342,7 @@ class blob_backend
 
     virtual void trim(std::size_t newLen) = 0;
 
-    virtual session_backend &get_session_backend() = 0;
+    virtual session_backend &get_session_backend() const = 0;
```

Comment:
This one seems wrong to me as well: `const` accessor should return const reference.

Reply:
`const` functions just mean that no member variable is modified.